docs/modules/ROOT/pages/reference/hierarchy.adoc

```diff
@@ -0,0 +1,89 @@
+= Configure the inventory hierarchy
```

Comment:
```suggestion
= Configure the Inventory Hierarchy
```

Reply:
The Microsoft style guide (which we're using according to our documentation) says to use sentence case for titles (cf. https://docs.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings)

Reply:
The vale checker does not assert this but the Microsoft style guide asks for sentence case in titles.